packages/core/test/dialog/dialogTests.tsx

```diff
@@ -64,6 +64,69 @@ describe("<Dialog>", () => {
         document.body.removeChild(container);
     });
 
+    describe("accessibility support", () => {
+        const mountDefaults = (className: string) => {
+            const container = document.createElement("div");
+            document.body.appendChild(container);
+            mount(
+                <Dialog
+                    className={className}
+                    isOpen={true}
+                    portalContainer={container}
+                    aria-labelledby="dialog-title"
+                    aria-describedby="dialog-description"
+                >
+                    {createDialogContents()}
+                </Dialog>,
+            );
+
+            return () => document.body.removeChild(container);
```

Comment:
similar feedback here... I think it would be more idiomatic to put this in a `afterEach` block...


```
afterEach(() => {
  if (container != null) {
    document.body.removeChild(container);
  }
});
```